core/session.ts

```diff
@@ -463,6 +469,32 @@ export class Session {
     });
   }
 
+  private getTablePrefixWithUnderscore() {
+    return !!this.config.tablePrefix ? `${this.config.tablePrefix}_` : "";
+  }
+
+  private prependTablePrefix(actions: IActionProto[]) {
```

Comment:
This code is basically the same as the appendSuffix code, can we put this all into one method plz?